lib/active_model_serializers/rspec_matchers/schema.rb

```diff
@@ -0,0 +1,24 @@
+require 'active_model_serializers/test/schema'
+
+module ActiveModelSerializers
+  module RSpecMatchers
+    module Schema
+      RSpec::Matchers.define :have_valid_schema do |message|
```

Comment:
Let's use RSpec default message handling instead. the only difference is that the custom message is not prepended to the original message, it substitutes it.

For example in _ActiveModelSerializers::Test::SchemaTest::MyController test_that_raises_error_with_a_custom_message_with_a_invalid_schema_

Instead of

`oh boy the show is broken: #/name: failed schema #/properties/name: For 'properties/name', \"Name 1\... schema #/properties/description: For 'properties/description', \"Description 1\" is not a boolean.`

we simply get the custom message:

`oh boy the show is broken`